docs/src/whatsnew/latest.rst

```diff
@@ -69,6 +69,8 @@ This document explains the changes made to Iris for this release
 
 #. `@rcomer`_ clarified instructions for updating gallery tests. (:pull:`5100`)
 
+#. `@trexfeathers`_ fixed some broken links. (:pull:`5109`)
+
```

Comment:
Could maybe be slightly more specific?

Reply:
17303d78ba1660eb4f115c5932d355abc191b7f9